spring-vault-core/src/main/java/org/springframework/vault/authentication/LdapAuthenticationOptions.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.vault.authentication;
+
+/**
+ * @author Mikhael Sokolov
+ */
+public class LdapAuthenticationOptions extends UsernamePasswordAuthenticationOptions {
+
+    private LdapAuthenticationOptions(String username, String password) {
+        super(username, password);
+    }
+
+    @Override
+    public String getPath() {
+        return String.format("auth/ldap/login/%s", username);
```

Comment:
Let's configure the path through options.

Reply:
done